addOns/paramminer/src/main/javahelp/org/zaproxy/addon/paramminer/resources/help/contents/simple.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Simple Example
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Simple Example</H1>
```

Comment:
filename and content could be updated/outlined for param miner.

Reply:
I can keep it blank right? I mean the body can have no content yet because I haven't yet defined these components :)

Reply:
Like I can add the bare minimum description and stuffs for now

Reply:
Blank or minimal, whatever.